pkg/controller/storagecluster/storagecluster_controller_test.go

```diff
@@ -3,9 +3,10 @@ package storagecluster
 import (
 	"context"
 	"fmt"
-	consolev1 "github.com/openshift/api/console/v1"
 	"testing"
 
+	consolev1 "github.com/openshift/api/console/v1"
+
```

Comment:
Is this auto-formatting? Either way it should go with the other OCP imports below.